articles/react-native-firebase-image-labelling/index.md

```diff
@@ -0,0 +1,425 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/react-native-image-labeling-with-firebase/
+title: Image Labeling Using Firebase ML in a Non-Expo React Native Application 
+description: This tutorial will give readers a detailed guide on how to implment image labeling using Firebase's ML kit in a Non-Expo React Native appliaction.
+author: mohan-raj
+date: 2021-01-07T00:00:00-15:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/react-native-image-labeling-with-firebase/hero.jpg
+    alt: React Native Image Labeling With Firebase Hero Image
+---
+Image labeling gives you insight into the content of images. In this tutorial, we will be building a Non-Expo React Native application to label the image provided using Firebase's machine learning kit.
```

Comment:
This section should not contain this `can't` but rather `cannot`. affected word is `Firebase's` Ref peer review documentation